CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/blaze/BlazePluginUtilities.java

```diff
@@ -48,12 +63,11 @@ static BitSet verticesParam(final PluginParameters parameters) {
         } else if (vParam.getClass() == BitSet.class) {
             vertices = (BitSet) vParam;
         } else {
-            final List<Integer> vertexList = (List<Integer>) vParam;
+            final List<Object> vertexList = (List<Object>) vParam;
```

Comment:
@algol60 was the reason you changed this to `Object` because `toInt()` expects an Object?

Reply:
I answered my own question...going to merge this. Thank you again @algol60.